doc/tutorial/get_started.md

```diff
@@ -57,11 +57,11 @@ Complete the following steps to install and initialise LXD:
 
 1. If LXD is already installed, enter the following command to update it:
 
-       sudo snap refresh lxd
+       sudo snap refresh lxd --channel=5.21/stable
 
    Otherwise, enter the following command to install LXD:
 
-       sudo snap install lxd
+       sudo snap install lxd --channel=5.21/stable
```

Comment:
LTS will always be set to the default track so it shouldn't be necessary to include it for `snap install`.

Reply:
True. But if you want a specific lts series that is compatible it would be worth making it explicit.